jenkins/gce-custom-images/prepare_gce_snapshot.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+
+set -e -x
+
+#
+# This script sets up a custom GCE snapshot for VMs provisioned by Leonardo.
```

Comment:
can we rephrase this to `sets up a snapshot of boot disk` or something (there's nothing "custom" about the snapshot in that all snapshots are custom made anyway)....just don't want to confuse ppl with custom image.

Reply:
Sure. I can change it in the main PR.